### ISSUE TITLE
NVDA focus jumps to the OK button immediately after opening the about dialog.

### ISSUE THREAD

**Issue by jcoffin1** [nvaccess/nvda#19285]
### Brief summary

When pressing NVDA+N and H then A to open the about dialog, NVDA focus jumps to the OK button in braille but speech reads out the entire dialog 

### Steps to reproduce

Connect a braille display if you have one. 
Press NVDA+n to open the NVDA menu. 
Press H to get to the help menu. 
Press A to get to the about dialog. 

When the about dialog is open, you'll see "OK" on your braille display. 

### Actual behavior

          Current: NVDA braille focus is at the OK button causing you to have to pan to the left all the way to the top of the dialog to get the information you want such as NVDA version number. 

### Expected behavior

Expected: NVDA focus should be at the top of the about dialog. 

### NVDA type

installed copy

### NVDA version

2025.3.2 RC 1 

### Windows version

Version	10.0.26100 Build 26100

### Other information about your system

System Manufacturer	Dell Inc.
System Model	Precision 5540
System Type	x64-based PC
Processor	Intel(R) Core(TM) i7-9850H CPU @ 2.60GHz, 2592 Mhz, 6 Core(s), 12 Logical Processor(s)
BIOS Version/Date	Dell Inc. 1.39.0, 8/8/2025
SMBIOS Version	3.2

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by Adriani90** [comment]
Probably this impacts many dialogs in general. nVDA cannot focus the top of the dialog, because the informationa bout NVDA version is not an interactive element.
You can enable the follow speech braille mode with nvda+alt+t, in which the NVDA version will be displayed on your braille device when NVDA speaks it.
Otherwise, this issue is actually requesting something like #258.

**Comment by seanbudd** [comment]
This is intended behaviour
